elrond-wasm-debug/src/testing_framework/contract_obj_wrapper.rs

```diff
@@ -35,16 +41,30 @@ where
     CB: ContractBase<Api = DebugApi> + CallableContract<DebugApi> + 'static,
     ContractObjBuilder: 'static + Copy + Fn(DebugApi) -> CB,
 {
-    pub fn new(_workspace_path: &str) -> Self {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        let mut current_dir = std::env::current_dir().unwrap();
+        current_dir.push(PathBuf::from_str("mandos/").unwrap());
```

Comment:
I think the final slash is not necessary, but doesn't hurt either

Reply:
iirc, PathBuff::push() doesn't work correctly without the slash, as it considers it an absolute path and overwrites the whole path with it.